oss-cloud-backend/handler.js

```diff
@@ -112,56 +91,46 @@ module.exports.updatePullRequests = async () => {
   let response;
   try {
     const results = await gitHubApiService.updatePullRequests();
-    response = {
-      statusCode: 200,
-      body: `${results} contributors updated`,
-    };
+    response = utility.generatesReponse(200, `${results} contributors updated`, false);
   } catch (error) {
     console.log('error in getPullRequests handler: ', error);
-    response = {
-      statusCode: 500,
-      body: JSON.stringify({
-        message: error.message,
-      }),
-    };
+    response = utility.generatesReponse(500, {
+      message: error.message,
+      success: false,
+    });
   }
   return response;
 };
 
 module.exports.getContributions = async (event) => {
   const {
-    sortBy, sortDesc, page, itemsPerPage, searchText, usernameSearch, repoSearch, titleSearch, dateFrom, dateTo, statusFilter,
-  } = event.queryStringParameters;
+    sortBy, sortDesc, page, itemsPerPage, searchText, usernameSearch,
+    repoSearch, titleSearch, dateFrom, dateTo, statusFilter,
+  } = event.queryStringParameters || {};
   let response;
   try {
     const [contributions, [contributionsLength]] = await databaseService.getContributionsPaging({
-      sortBy,
+      sortBy: typeof (sortBy) === 'string' ? sortBy : 'author',
       sortDesc: sortDesc === 'true',
-      page: parseInt(page, 10),
-      itemsPerPage: parseInt(itemsPerPage, 10),
-      searchText,
+      page: page ? parseInt(page, 10) : 1,
+      itemsPerPage: itemsPerPage ? parseInt(itemsPerPage, 10) : 13,
+      searchText: searchText || '',
       usernameSearch: usernameSearch === 'true',
       repoSearch: repoSearch === 'true',
       titleSearch: titleSearch === 'true',
-      dateFrom,
-      dateTo,
-      statusFilter,
+      dateFrom: dateFrom || '2000-01-01',
+      dateTo: dateTo || '2030-01-01',
```

Comment:
Probably, the better idea is to choose `now` for dateTo.

Reply:
Changed to `new Date()`.